src/containers/FormikForm/FormikGrepCodesContent.tsx

```diff
@@ -124,6 +137,11 @@ const FormikGrepCodesContent = ({
   return (
     <Fragment>
       <FormikFieldDescription description={t('form.grepCodes.description')} />
+      {failedGrepCodes && (
+        <FormikFieldHelp error>
+          <StyledErrorPreLine>{t('errorMessage.grepCodes')}</StyledErrorPreLine>
+        </FormikFieldHelp>
+      )}
```

Comment:
Har ikkje testa koden, men nokke sånt kan vise fram kodene som feila. Krever at du endrer teksten litt i phrases.
```suggestion
      {failedGrepCodes.length && (
        <FormikFieldHelp error>
          <StyledErrorPreLine>{`${t('errorMessage.grepCodes')}${failedGrepCodes.join(',')}`}</StyledErrorPreLine>
        </FormikFieldHelp>
      )}
```